caffeine/src/main/java/com/github/benmanes/caffeine/cache/BoundedLocalCache.java

```diff
@@ -441,11 +441,21 @@ protected void lazySetMainProtectedMaximum(long maximum) {
    * to eagerly evicts entries until the cache shrinks to the appropriate size.
    */
   @GuardedBy("evictionLock")
-  void setMaximum(long maximum) {
+  void setMaximum(long maximum){
+      setMaximum(maximum, PERCENT_MAIN);
+  }
+
+  /**
+   * Sets the maximum weighted size of the cache and the ration between eden and LFU.
```

Comment:
ration -> ratio

Reply:
Maybe the explanation should be less hand-wavy and explicitly say it defines the percentage given over to the LFU.